xarray/core/options.py

```diff
@@ -269,3 +271,73 @@ def __enter__(self):
 
     def __exit__(self, type, value, traceback):
         self._apply_update(self.old)
+
+
+def get_options():
+    """
+    Get options for xarray.
+
+    Parameters
```

Comment:
Shall we instead just refer the user to the docstring for `set_options`. That way it is only ever described in one place?

Reply:
I've just done this. I also added an additional parameter to the test to make sure that `get_options` is actually tracking `set_options` changes and not just matching because of a default value or so.